src/ui/frmpreferences.cpp

```diff
@@ -284,9 +310,18 @@ void frmPreferences::saveLanguages(QSettings *s)
     }
 }
 
-void frmPreferences::saveColorScheme(QSettings *s)
+void frmPreferences::saveAppearanceTab(QSettings *s)
 {
     s->setValue("Appearance/ColorScheme", ui->cmbColorScheme->currentData().toString());
+
+    QString font = ui->cmbFontFamilies->isEnabled() ? ui->cmbFontFamilies->currentFont().family() : "";
+    int size = ui->spnFontSize->isEnabled() ? ui->spnFontSize->value() : 0;
+
+    Editor::setGlobalFontFamily( font );
+    Editor::setGlobalFontSize( size );
+
+    s->setValue("overrideFontFamily", font);
+    s->setValue("overrideFontSize", size);
```

Comment:
These should be under `Appearance/OverrideFont*`, as happens with "ColorScheme" (they're both in the Appearance tab).